src/priest.c

```diff
@@ -580,8 +580,10 @@ register struct monst *priest;
             long pmoney = money_cnt(priest->minvent);
             if (pmoney > 0L) {
                 /* Note: two bits is actually 25 cents.  Hmm. */
-                pline("%s gives you %s for an ale.", Monnam(priest),
-                      (pmoney == 1L) ? "one bit" : "two bits");
+                pline("%s gives you %s %s%s for an ale.", Monnam(priest),
+                      (pmoney == 1L) ? "one" : "two",
+                      Hallucination ? "bitcoin" : "bit",
+                      (pmoney == 1L) ? "" : "s");
```

Comment:
No. Don't drag that clusterfuck into NetHack.
                                                         
We already have hallucinated currencies, how about:
`const char *bits = (Hallucination) ? currency(pmoney) : (pmoney == 1L) ? "bit" : "bits";`
`pline("%s gives you %s%s for an ale.", Monnam(priest), (pmoney == 1L) ? "one " : "two ", bits);`